documentation/source/user_section/tutorials/registration_to_template.rst

```diff
@@ -0,0 +1,376 @@
+     - Labels are produced for both vertebral levels and intervertebral discs.
+
+  #. Finally, the spinal cord and the labeled segmentation are both un-straightened, and the labels are saved to image files.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/instrumentation-missing-discs.png
+   :align: center
+   :figwidth: 400px
+
+   ``sct_label_vertebrae`` is able to label vertebral levels despite missing discs due to instrumentation.
+
+To apply the labeling algorithm, we use the following command:
+
+.. code:: sh
+
+   sct_label_vertebrae -i t2.nii.gz -s t2_seg.nii.gz -c t2 -qc ~/qc_singleSubj
+
+   # Input arguments:
+   #   - i: Input image
+   #   - c: Contrast of the input image
+   #   - qc: Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+
+   # Output files/folders:
+   #   - t2_seg_labeled.nii.gz: Image containing the labeled spinal cord. Each voxel of the segmented spinal cord is
+   #                            labeled with a vertebral level as though the vertebrae were projected onto the spinal
+   #                            cord. The convention for label values is C3-->3, C4-->4, etc.
+   #   - t2_seg_labeled_discs.nii.gz: Image containing single-voxel intervertebral disc labels (without the segmented
+   #                                  spinal cord). Each label is centered within the disc. The convention for label
+   #                                  values is C2/C3-->3, C3/C4-->4, etc. This file also contains additional labels
+   #                                  (such as the pontomedullary junction and groove), but these are not yet used.
+   #   - straight_ref.nii.gz: The straightened input image produced by the intermediate straightening step. Can be
+   #                          re-used by other SCT functions that need a straight reference space.
+   #   - warp_curve2straight.nii.gz: The 4D warping field that defines the transform from the original curved
+   #                                 anatomical image to the straightened image.
+   #   - warp_straight2curve.nii.gz: The 4D warping field that defines the inverse transform from the straightened
+   #                                 anatomical image back to the original curved image.
+   #   - straightening.cache: If sct_label_vertebrae is run another time, the presence of this file (plus
+   #                          straight_ref.nii.gz and the two warping fields) will cause the straightening step to be
+   #                          skipped, thus saving processing time.
+
+.. note::
+
+   If the labeling fails, you may also manually label the C2-C3 disc using ``sct_label_utils``, then re-run ``sct_label_vertebrae`` with this initialized image.
```

Comment:
Right now, this PR contains only a small comment on manual labelling. (I believe this is more-or-less what was in the SCT course, as well.)

Reply:
Done!